network-config.yaml

```diff
@@ -22,3 +22,16 @@ development:
     host: http://127.0.0.1
     id: goerli-fork
     name: goerli-fork
+    timeout: 120
+  - cmd: ./ganache.sh
+    cmd_settings:
+      accounts: 10
+      chain_id: 5
+      fork: http://127.0.0.1:7777
+      gas_limit: 12000000
+      mnemonic: brownie
+      port: 8545
+    host: http://127.0.0.1
+    id: local-fork
```

Comment:
what is the purpose of this local-fork?

Reply:
Just useful for development for mainnet because need separate fork with the contracts deployed. but should be removed after deployment on mainnet